gef-ui/templates/includes/submit/submit-to-checker-post-ukef-approval.njk

```diff
@@ -0,0 +1,12 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+<p class="govuk-heading-l" data-cy="submit-heading">
+  Submit
+</p>
+
+  {{ govukButton({
+    text: "Submit to be checked at your bank",
+    attributes: {
+      'data-cy': 'submit-button'
```

Comment:
could we change this to be more explicit as to what it submits? it will be easier to read when we select it in the tests

Reply:
Do you mean change the filename @IgnatG

Reply:
nop. 
this value here 
` 'data-cy': 'submit-button'`
so I was wondering if we can change this `submit-button` text, into something more meaningful :)

Reply:
Done